Kiss.Bff/Extern/Elasticsearch/ElasticsearchController.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Kiss.Bff.Extern.ElasticSearch
+{
+    /// <summary>
+    /// Elasticsearch proxy controller with request and response body transformation
+    /// </summary>
+    [Route("api/elasticsearch")]
+    [ApiController]
+    public class ElasticsearchController : ControllerBase
+    {
+        private readonly HttpClient _httpClient;
+        private readonly string _elasticsearchBaseUrl;
+        private readonly string _elasticsearchUsername;
+        private readonly string _elasticsearchPassword;
+        private readonly ILogger<ElasticsearchController> _logger;
+
+        public ElasticsearchController(
+            HttpClient httpClient,
+            IConfiguration configuration,
+            ILogger<ElasticsearchController> logger)
+        {
+            // Load configuration
+            _elasticsearchBaseUrl = configuration["ELASTIC_BASE_URL"] ?? throw new InvalidOperationException("ELASTIC_BASE_URL not configured");
+            _elasticsearchUsername = configuration["ELASTIC_USERNAME"] ?? throw new InvalidOperationException("ELASTIC_USERNAME not configured");
+            _elasticsearchPassword = configuration["ELASTIC_PASSWORD"] ?? throw new InvalidOperationException("ELASTIC_PASSWORD not configured");
+
+            _httpClient = httpClient;
+            _httpClient.BaseAddress = new Uri(_elasticsearchBaseUrl);
+            _logger = logger;
+        }
+
+        [HttpPost("{index}/_search")]
+        [Authorize(Policy = Policies.KcmOrKennisbankPolicy)]
+        public async Task<IActionResult> Search([FromRoute] string index)
```

Comment:
Je kan het jezelf een stuk makkelijker maken:

```suggestion
        public async Task<IActionResult> Search([FromRoute] string index, [FromBody] JsonObject elasticqQuery, CancellationToken cancellationToken)
```